src/Oracles/Config/Setting.hs

```diff
@@ -133,6 +138,16 @@ osxHost = anyHostOs ["darwin"]
 windowsHost :: Action Bool
 windowsHost = anyHostOs ["mingw32", "cygwin32"]
 
+-- https://github.com/ghc/ghc/blob/master/mk/config.mk.in#L241
+relocatableBuild :: Action Bool
+relocatableBuild = windowsHost
+
+installDir :: Action String
+installDir = (++ " -m 755 -d") <$> setting Install
```

Comment:
An alternative way is to add `install-dir` and `install-data` into `cfg/system.config.in`, e.g.:
```
install-dir = @INSTALL@ -m 755 -d
```
I'm not sure which way is better. Going through `cfg/system.config.in` may be a bit more flexible if some of these settings will end up requiring more Autoconf magic.